README.md

```diff
@@ -1,29 +1,13 @@
-# Adobe I/O CodeLabs template
+# Build a Firefly App for Customer Profiles using Adobe Campaign Standard API
 
-## Getting started 
+[Project Firefly](https://github.com/AdobeDocs/project-firefly) is a complete framework that enables enterprise developers to build and deploy custom web applications that extend Adobe Experience Cloud solutions and run on Adobe infrastructure. It leverages modern technologies (JAM stack, serverless computing, Node, and React) and ensures best practices when building applications (event-driven architecture, microservices, continuous integration, and delivery).  
 
-Click on the "Use this template" button to copy the template to your namespace. 
+[Adobe Campaign Standard (ACS)](https://www.adobe.com/marketing/campaign.html) provides a platform for designing and executing digital marketing campaigns and provides an environment for visual campaign orchestration.  
 
-The codelab entry point is the [README](README.md). You can put your codelab content including assets under the [lessons](/lessons) directory. The codelab [manifest.json](manifest.json) holds the navigation information and more. Link your lessons in that file in order to render the navigation.
+[Campaign Standard APIs](https://docs.adobe.com/content/help/en/campaign-standard/using/working-with-apis/about-campaign-standard-apis/about-campaign-standard-apis.html) are aimed at letting you create integrations for Adobe Campaign Standard and build your own ecosystem by interfacing Adobe Campaign Standard with the panel of technologies that you use.
 
-## Preview codelab locally
+In this lab, you will learn how to build a simple web app listing customer profiles and allowing personalized email promotions through Campaign Standard API.  
 
-The template codelab is based on the Project Helix https://www.project-helix.io/. Follow instructions as explained.
+For your convenience, we provide a complete solution of this codelab at [AdobeDocs/adobeio-awesomeapps-customers-dashboard](https://github.com/AdobeDocs/adobeio-awesomeapps-customers-dashboard).  
```

Comment:
Q: should we always 1) provide the source code of a codelab project n a separate repository like here or 2) rather add it to the codelab repo itself ?

If 1) should we stick to repo name : `adobeio-awesomeapps-X` ?

Reply:
I strongly prefer 1 so that people can view them without going through the codelab lesson.

Not sure about the naming convention of awesome apps, currently I just name it similarly to `adobeio-codelabs-X`. We could use `adobeio-awesome-X` as well...

Reply:
it's already decided to "AdobeDocs/adobeio-awesomeapps-customers-dashboard." hence ok to me.